README.md

```diff
@@ -100,6 +100,12 @@ Example for full-enclosure analysis:
 python pyV2DL3/script/v2dl3_for_Eventdisplay.py --full-enclosure -f 64080.anasum.root [Effective Area File] ./outputdir/64080.anasum.fits
 ```
 
+The run having their parameters outside but close to corresponding IRF range can be run with the one of the following two commandline parameters: 
+
+- `--force_extrapolation`: This option extrapolates the IRF at the run parameter value.
```

Comment:
Mention type of extrapolation: extrapolates linearly (is it linearly?) beyond the edges of the IRF axes' to the value of the ....

Reply:
Add a word of caution, is this also extrapolates to very far values outside of the range of the IRF axes.